src/pages/components/progress-bar/style.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Progress bar
+description:
+  A progress bar provides feedback about the duration and progression of a
+  process.
+tabs: ['Usage', 'Style']
+---
+
+## Color
+
+| Element       | Property   | Color token           |
+| ------------- | ---------- | --------------------- |
+| Label         | Text color | `$text-primary`       |
+| Helper text   | Text color | `$text-helper`        |
+| Track         | Background | `$border-subtle`      |
+| Bar: active   | Background | `$border-interactive` |
+| Bar: success  | Background | `$support-success`    |
+| Icon: success | Fill       | `$support-success`    |
+| Bar: error    | Background | `$support-error`      |
+| Icon: error   | Fill       | `$support-error`      |
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar status colors](images/progress-bar-style-1.png)
+
+<Caption>Status colors</Caption>
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={12}>
+
+![Progress bar status colors](images/progress-bar-style-2.png)
+
+<Caption>Progress bar colors for light themes</Caption>
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={12}>
+
+![Progress bar status colors](images/progress-bar-style-3.png)
+
+<Caption>Progress bar colors for dark themes</Caption>
+
+</Column>
+</Row>
+
+## Typography
+
+The width and height of a popover container can vary depending on the amount of
```

Comment:
This sentence can be removed. I think it was a carry over from a different component page possibly.